lxd/device/device_utils_unix.go

```diff
@@ -550,3 +550,45 @@ func unixDeviceDeleteFiles(s *state.State, devicesPath string, typePrefix string
 
 	return nil
 }
+
+// unixValidDeviceNum validates the major and minor numbers for a UNIX device.
+func unixValidDeviceNum(value string) error {
+	if value == "" {
+		return nil
+	}
+
+	_, err := strconv.ParseUint(value, 10, 32)
+	if err != nil {
+		return fmt.Errorf("Invalid value for a UNIX device number")
+	}
+
+	return nil
+}
+
+// unixValidUserID validates the UNIX UID and GID values for ownership.
+func unixValidUserID(value string) error {
```

Comment:
This is identical to IsUnixUserID so should go away

Reply:
Oh, I see, you want to move them to be only in the device package, I guess that's fine.